sample-apps/java/client/devserver-client-sample/src/main/java/clientsample/YourWorker.java

```diff
@@ -0,0 +1,29 @@
+package clientsample;
+
+import io.temporal.client.WorkflowClient;
+import io.temporal.worker.Worker;
+import io.temporal.worker.WorkerFactory;
+import io.temporal.serviceclient.WorkflowServiceStubs;
+
+public class YourWorker {
+    public static void initiateWorker(String[] args) {
+        // Create a stub that accesses a Temporal Service on the local development machine
+        WorkflowServiceStubs service = WorkflowServiceStubs.newLocalServiceStubs();
+
+        // A Workflow Client can start, Signal, and Query a Workflow Execution, etc
```

Comment:
While it's true that you can start, Signal, and Query a Workflow Execution using a Client, the Worker is not doing any of those things. I'd change this comment to something like "The Worker uses the Client to communicate with the Temporal Service" and then put the comment you have here in the `YourCallerApp`, which is where you have a Client that starts the Workflow.

Reply:
Rephrasing. Thank you!